index.js

```diff
@@ -475,3 +476,52 @@ export async function whoami() {
   const client = await getClient()
   console.log(client.did())
 }
+
+/**
+ * @param {object} [opts]
+ * @param {boolean} [opts.human]
+ * @param {boolean} [opts.json]
+ */
+export async function usageReport(opts) {
+  const client = await getClient()
+  const now = new Date()
+  const period = {
+    // we may not have done a snapshot for this month _yet_, so get report from last month -> now
+    from: startOfLastMonth(now),
+    to: now
```

Comment:
should we log the period we are tracking?

Reply:
No it's a report as of _now_.

If we were graphing the data then yes but we are not in the CLI.